### ISSUE TITLE
visor instructions - attachment screws unclear

### ISSUE THREAD

**Issue by rsc** [hybridgroup/gopherbot#3]
At the [attach circuit to backpack step](https://github.com/hybridgroup/gopherbot/blob/master/assembly/visor.md#attach-circuit-playground-express-to-backpack) the instructions are a little unclear about which spots on the board to attach to the backpack. It says to use four 10mm screws but there are six possible holes that could be used. (There are also six 10mm screws, but I haven't gotten far enough to know if the two other screws are going to be used for something else.)

At the next step, [attach ribbon cable red and black/white wires](https://github.com/hybridgroup/gopherbot/blob/master/assembly/visor.md#attach-ribbon-cable-red-and-blackwhite-wires-to-circuit-playground-express-and-backpack), I assume that you are supposed to attach them between the board and the washer, so that means you have to put two of the 10mm screws from the previous step in those spots. (I had not chosen both of those spots in the previous step, and I had to backtrack.)

The text in that next step is also inconsistent (or I am confused). It says "Take 2 of the 6mm machine screws and 2 of the M3 nuts" but then says "using the 10mm machine screw and one of the M3 nuts" twice. It seems like you should reuse the 10mm screw assemblies from the previous step, so that there wouldn't be anything new in this step? Or maybe you are supposed to finish off the six posts in this step?

Anyway, having fun but the docs could be a little clearer at this point. Thanks!

**Comment by deadprogram** [comment]
Yes, I was trying to update those docs unsuccessfully it turns out, to try to change how the washers were to be placed.

In short, you are correct that there are 6 screws used for the standoffs to hold the backpack to the board. 2 of those screws are used to connect the red and black/white terminals. The other 4 should have washers on the screw on the back side of the board in order to have the same spacing between the board and the standoffs to match the space taken up by the red and black/white fork connectors.

I am going to make another attempt at making those instructions a bit clearer. Probably i should have left my original step order for this, I was trying to improve it, but ended up making it less clear.

**Comment by deadprogram** [comment]
I just updated the assembly instructions to try to address this issue @rsc please let me know what you think. Thank you.
